File: python/Level/5/5622.py
w = ["ABC", "DEF", "GHI", "JKL", "MNO", "PQRS", "TUV", "WXYZ"]
s = input()
result = 0

for i in range(len(s)):
    for j in w:
        if s[i] in j:
            result += w.index(j) + 3
            
print(result)

File: cpp/Level/5/5622.cpp
#include <iostream>
#include <string>

using namespace std;

int main() {
    string w[] = {"ABC", "DEF", "GHI", "JKL", "MNO", "PQRS", "TUV", "WXYZ"};
    string s;
    int result = 0;

    cin >> s;

    for (int i = 0; i < s.length(); i++) {
        for (int j = 0; j < 8; j++) {
            if (w[j].find(s[i]) != string::npos) {
                result += j + 3;
            }
        }
    }

    cout << result;

    return 0;
}